Sources/Complex/Complex.swift

```diff
@@ -313,7 +313,7 @@ extension Complex: Hashable {
     // Identify all numbers with either component non-finite as a single
     // "point at infinity".
     guard a.isFinite || b.isFinite else { return true }
-    // For finite numbers, equality is defined componentwise. Cases where
+    // For finite numbers, equality is defined component wise. Cases where
```

Comment:
This one is correct as written.

Reply:
Ah, American English vs Brittish English. I'm used to the latter! Will make sure to remember this for the future :)